EnglishFilter/sections/general_extensions.txt

```diff
@@ -528,6 +528,8 @@ shemalestube.com#%#//scriptlet('prevent-window-open')
 dotabuff.com#%#//scriptlet("abort-on-property-write", "pbjs")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/83227
 thepiratebay.org#%#//scriptlet('remove-attr', 'href', 'header a[href^="https://exchange."]')
+! https://github.com/AdguardTeam/AdguardFilters/issues/95401
+thepiratebay.org#%#//scriptlet('remove-attr', 'href', 'nav > section:first-of-type > a')
```

Comment:
Hello @marysaotomeprog 
We usually avoid such selectors as "first-of-type", "n-th child" etc. Let's just replace the selector in the old rule:

`thepiratebay.org#%#//scriptlet('remove-attr', 'href', 'header a[href^="https://exchange."]')`

I propose this one: `#home > header  a[href^="https://"]`